src/printer/tir_text_printer.cc

```diff
@@ -149,26 +149,28 @@ Doc TIRTextPrinter::PrintPrimFunc(const PrimFunc& prim_func) {
       buffer_map_doc.push_back(Print(v) << ": " << Print(buf));
     }
     doc << Doc::Indent(
-        2, Doc::NewLine() << "buffer_map = {" << PrintSep(buffer_map_doc, Doc::Text(", ")) << "}");
+        2, NewLine() << "buffer_map = {" << PrintSep(buffer_map_doc, Doc::Text(", ")) << "}");
```

Comment:
This calls `virtual TIRTextPrinter::NewLine` instead of `Doc::NewLine` so the debug text printer can flush the buffer of statements, so really there should be no direct `Doc::NewLine` calls